be1-go/internal/popserver/hub.go

```diff
@@ -56,6 +56,7 @@ func (h *Hub) Start() {
 	go func() {
 		utils.LogInfo("start the Hub")
 		for {
+			utils.LogInfo("waiting for a new message")
```

Comment:
When you clean up logs, this would have to be a `Debug`. This will spam like crazy when the server is active